tests/test_sklearn_pipeline.py

```diff
@@ -471,7 +478,6 @@ def test_pipeline_column_transformer_drop_noweights(self):
             model.predict_proba(X_test), torch_model.predict_proba(X_test.values), rtol=1e-06, atol=1e-06,
         )
 
-    @unittest.skipIf(ColumnTransformer is None, reason="ColumnTransformer not available in 0.19")
```

Comment:
Did you pull this b/c  it's now part of `_parse_sklearn_stacking`?  or do these tests just happen to not hit it?

Reply:
We only support Sklearn >= 0.21.3 so this is redundant.